### PR TITLE
feat: report quotes

### PR DESCRIPTION
**Pull request by Sanix-Darker**
# What
- added a github action for sending programming quotes

### CODE REVIEW

File: .github/workflows/report_quotes.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+
+name: telegram quote report
+on:
+  schedule:
+    - cron: "* */10 * * *"
```
Comment (elhmn) [inline .github/workflows/report_quotes.yaml:5]: Did you mean "0 10 * * *" ?

**Review by elhmn — COMMENTED** [review]
Great Idea, I feel like the cron can be adjusted a bit, let me know what you think.

File: .github/workflows/report_quotes.yaml (hunk 1 above)
Comment (Sanix-Darker) [reply .github/workflows/report_quotes.yaml:5]: Nope... i meant each 10hours, looks too much ? 🤔

Comment (Sanix-Darker) [reply .github/workflows/report_quotes.yaml]: updated ! 👍🏾

Comment (elhmn) [reply .github/workflows/report_quotes.yaml]: I see in this you probably wanted to say this "0 */10 * * *" 👌

**Pull request merged**